packages/dx-demo-shell/generate-demos.js

```diff
@@ -121,18 +136,31 @@ const overrideFileIfChanged = (filename, data) => {
 };
 const createFromTemplate = (sourceFilename, outputFilename, data) => {
   const source = fs.readFileSync(sourceFilename, 'utf-8');
-  mustache.tags = ['<%', '%>'];
   const output = mustache.render(source, data);
   overrideFileIfChanged(outputFilename, output);
   cancelFileRemoving(outputFilename);
 };
+const findSplitter = (content, index) => {
+  const k = content.indexOf(' from ', index + 1);
```

Comment:
What about using `lastIndexOf` instead of `indexOf`?

Reply:
It is slightly more dangerous.
Searching from the beginning of a file chance that `__from__` is part of `import` statement is quite high.
Searching from the ending of a file `__from__` may be part of some title value.
```jsx
<SomePlugin title="Some data from some source" />
```

However, if there is no `import` section and there is some title with *from* word current search would not work as well.

----
Current search needs improvement.
And it is `+1` for slitter tag option.